modules/openapi-generator/src/main/resources/C-libcurl/model-body.mustache

```diff
@@ -344,7 +344,12 @@ cJSON *{{classname}}_convertToJSON({{classname}}_t *{{classname}}) {
     {{/required}}
     {{^required}}
     {{^isEnum}}
+    {{#isString}}
```

Comment:
Would maybe the isInteger/isBoolean check be safer, similar to:
https://github.com/ityuhui/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/C-libcurl/api-body.mustache#L216

Reply:
Yeah sounds good.